
 Network guilhermegules-payflow-node_backend  Creating
 Network guilhermegules-payflow-node_backend  Created
 Container db  Creating
 Container redis  Creating
 Container redis  Created
 Container db  Created
 Container worker  Creating
 Container api-1  Creating
 Container api-2  Creating
 Container worker  Created
 Container api-1  Created
 Container api-2  Created
 Container nginx  Creating
 Container nginx  Created
Attaching to api-1, api-2, db, nginx, redis, worker
redis   | 1:C 24 Jul 2025 19:40:06.007 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
redis   | 1:C 24 Jul 2025 19:40:06.007 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis   | 1:C 24 Jul 2025 19:40:06.007 * Redis version=7.4.5, bits=64, commit=00000000, modified=0, pid=1, just started
redis   | 1:C 24 Jul 2025 19:40:06.007 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
redis   | 1:M 24 Jul 2025 19:40:06.007 * monotonic clock: POSIX clock_gettime
redis   | 1:M 24 Jul 2025 19:40:06.008 * Running mode=standalone, port=6379.
redis   | 1:M 24 Jul 2025 19:40:06.008 * Server initialized
redis   | 1:M 24 Jul 2025 19:40:06.009 * Ready to accept connections tcp
db      | The files belonging to this database system will be owned by user "postgres".
db      | This user must also own the server process.
db      | 
db      | The database cluster will be initialized with locale "en_US.utf8".
db      | The default database encoding has accordingly been set to "UTF8".
db      | The default text search configuration will be set to "english".
db      | 
db      | Data page checksums are disabled.
db      | 
db      | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db      | creating subdirectories ... ok
db      | selecting dynamic shared memory implementation ... posix
db      | selecting default max_connections ... 100
db      | selecting default shared_buffers ... 128MB
db      | selecting default time zone ... UTC
db      | creating configuration files ... ok
db      | running bootstrap script ... ok
db      | sh: locale: not found
db      | 2025-07-24 19:40:08.199 UTC [35] WARNING:  no usable system locales were found
db      | performing post-bootstrap initialization ... ok
db      | syncing data to disk ... ok
db      | 
db      | 
db      | Success. You can now start the database server using:
db      | 
db      |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db      | 
db      | initdb: warning: enabling "trust" authentication for local connections
db      | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db      | waiting for server to start....2025-07-24 19:40:10.214 UTC [41] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
db      | 2025-07-24 19:40:10.219 UTC [41] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db      | 2025-07-24 19:40:10.238 UTC [44] LOG:  database system was shut down at 2025-07-24 19:40:09 UTC
db      | 2025-07-24 19:40:10.252 UTC [41] LOG:  database system is ready to accept connections
db      |  done
db      | server started
db      | CREATE DATABASE
db      | 
db      | 
db      | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
db      | CREATE TABLE
db      | CREATE INDEX
db      | 
db      | 
db      | waiting for server to shut down....2025-07-24 19:40:10.625 UTC [41] LOG:  received fast shutdown request
db      | 2025-07-24 19:40:10.633 UTC [41] LOG:  aborting any active transactions
db      | 2025-07-24 19:40:10.636 UTC [41] LOG:  background worker "logical replication launcher" (PID 47) exited with exit code 1
db      | 2025-07-24 19:40:10.638 UTC [42] LOG:  shutting down
db      | 2025-07-24 19:40:10.645 UTC [42] LOG:  checkpoint starting: shutdown immediate
db      | 2025-07-24 19:40:10.785 UTC [42] LOG:  checkpoint complete: wrote 925 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.034 s, sync=0.082 s, total=0.148 s; sync files=306, longest=0.057 s, average=0.001 s; distance=4260 kB, estimate=4260 kB
db      | 2025-07-24 19:40:10.793 UTC [41] LOG:  database system is shut down
db      |  done
db      | server stopped
db      | 
db      | PostgreSQL init process complete; ready for start up.
db      | 
db      | 2025-07-24 19:40:10.859 UTC [1] LOG:  starting PostgreSQL 15.13 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
db      | 2025-07-24 19:40:10.859 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db      | 2025-07-24 19:40:10.859 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db      | 2025-07-24 19:40:10.892 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db      | 2025-07-24 19:40:10.982 UTC [65] LOG:  database system was shut down at 2025-07-24 19:40:10 UTC
db      | 2025-07-24 19:40:10.989 UTC [66] FATAL:  the database system is starting up
db      | 2025-07-24 19:40:10.995 UTC [1] LOG:  database system is ready to accept connections
nginx   | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx   | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx   | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx   | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx   | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx   | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx   | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx   | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx   | /docker-entrypoint.sh: Configuration complete; ready for start up
api-1   | 
api-1   | > rinha-payflow@1.0.0 start:api
api-1   | > node dist/server.js
api-1   | 
api-2   | 
api-2   | > rinha-payflow@1.0.0 start:api
api-2   | > node dist/server.js
api-2   | 
worker  | 
worker  | > rinha-payflow@1.0.0 start:worker
worker  | > node dist/worker/worker.js
worker  | 
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:18 +0000] "GET /payments-summary HTTP/1.1" 502 157 "-" "curl/7.88.1"
nginx   | 2025/07/24 19:40:18 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://172.20.0.4:3000/payments-summary", host: "localhost:9999"
nginx   | 2025/07/24 19:40:18 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://172.20.0.6:3000/payments-summary", host: "localhost:9999"
api-2   | [dotenv@17.2.0] injecting env (0) from .env (tip: üîê encrypt with dotenvx: https://dotenvx.com)
api-1   | [dotenv@17.2.0] injecting env (0) from .env (tip: ‚öôÔ∏è  load multiple .env files with { path: ['.env.local', '.env'] })
api-2   | {"level":30,"time":1753386019302,"pid":18,"hostname":"f8f1c3c57cbd","msg":"Server listening at http://127.0.0.1:3000"}
api-2   | Server is running :: http://127.0.0.1:3000
api-2   | {"level":30,"time":1753386019303,"pid":18,"hostname":"f8f1c3c57cbd","msg":"Server listening at http://172.19.0.4:3000"}
api-2   | {"level":30,"time":1753386019303,"pid":18,"hostname":"f8f1c3c57cbd","msg":"Server listening at http://172.20.0.6:3000"}
api-1   | {"level":30,"time":1753386019306,"pid":18,"hostname":"7580ff6a71d1","msg":"Server listening at http://127.0.0.1:3000"}
api-1   | Server is running :: http://127.0.0.1:3000
api-1   | {"level":30,"time":1753386019307,"pid":18,"hostname":"7580ff6a71d1","msg":"Server listening at http://172.20.0.4:3000"}
api-1   | {"level":30,"time":1753386019307,"pid":18,"hostname":"7580ff6a71d1","msg":"Server listening at http://172.19.0.5:3000"}
nginx   | 2025/07/24 19:40:23 [error] 28#28: *4 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://backend/payments-summary", host: "localhost:9999"
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:23 +0000] "GET /payments-summary HTTP/1.1" 502 157 "-" "curl/7.88.1"
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:28 +0000] "GET /payments-summary HTTP/1.1" 502 157 "-" "curl/7.88.1"
nginx   | 2025/07/24 19:40:28 [error] 28#28: *5 no live upstreams while connecting to upstream, client: 172.20.0.1, server: , request: "GET /payments-summary HTTP/1.1", upstream: "http://backend/payments-summary", host: "localhost:9999"
api-2   | {"level":30,"time":1753386033419,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-1","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":59072},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:33 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-2   | {"level":50,"time":1753386033420,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-1","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":59072},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-2   | {"level":30,"time":1753386033425,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-1","res":{"statusCode":500},"responseTime":5.5004401206970215,"msg":"request completed"}
api-2   | {"level":30,"time":1753386038434,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-2","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":59074},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:38 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-2   | {"level":50,"time":1753386038434,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-2","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":59074},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-2   | {"level":30,"time":1753386038434,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-2","res":{"statusCode":500},"responseTime":0.6621518135070801,"msg":"request completed"}
api-1   | {"level":30,"time":1753386043447,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-1","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":37862},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:43 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-1   | {"level":50,"time":1753386043450,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-1","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":37862},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-1   | {"level":30,"time":1753386043454,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-1","res":{"statusCode":500},"responseTime":5.855439901351929,"msg":"request completed"}
api-2   | {"level":30,"time":1753386048463,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-3","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":44828},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:48 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-2   | {"level":50,"time":1753386048463,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-3","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":44828},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-2   | {"level":30,"time":1753386048463,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-3","res":{"statusCode":500},"responseTime":0.5776610374450684,"msg":"request completed"}
api-1   | {"level":30,"time":1753386053473,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-2","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":51016},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:53 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-1   | {"level":50,"time":1753386053473,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-2","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":51016},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-1   | {"level":30,"time":1753386053474,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-2","res":{"statusCode":500},"responseTime":0.630640983581543,"msg":"request completed"}
api-2   | {"level":30,"time":1753386058484,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-4","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":49802},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:40:58 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-2   | {"level":50,"time":1753386058484,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-4","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":49802},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-2   | {"level":30,"time":1753386058484,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-4","res":{"statusCode":500},"responseTime":0.5745301246643066,"msg":"request completed"}
api-1   | {"level":30,"time":1753386063495,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-3","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":44350},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:41:03 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-1   | {"level":50,"time":1753386063496,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-3","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":44350},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-1   | {"level":30,"time":1753386063496,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-3","res":{"statusCode":500},"responseTime":0.8565058708190918,"msg":"request completed"}
api-2   | {"level":30,"time":1753386068509,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-5","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":53192},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:41:08 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-2   | {"level":50,"time":1753386068510,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-5","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":53192},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-2   | {"level":30,"time":1753386068511,"pid":18,"hostname":"f8f1c3c57cbd","reqId":"req-5","res":{"statusCode":500},"responseTime":1.2732479572296143,"msg":"request completed"}
api-1   | {"level":30,"time":1753386073521,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-4","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":40632},"msg":"incoming request"}
nginx   | 172.20.0.1 - - [24/Jul/2025:19:41:13 +0000] "GET /payments-summary HTTP/1.1" 500 81 "-" "curl/7.88.1"
api-1   | {"level":50,"time":1753386073522,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-4","req":{"method":"GET","url":"/payments-summary","host":"localhost","remoteAddress":"172.20.0.7","remotePort":40632},"res":{"statusCode":500},"err":{"type":"RangeError","message":"Invalid time value","stack":"RangeError: Invalid time value\n    at Date.toISOString (<anonymous>)\n    at Object.getPaymentSummary (/app/dist/services/payment-service.js:21:95)\n    at preHandlerCallbackInner (/app/node_modules/fastify/lib/handleRequest.js:161:24)\n    at preHandlerCallback (/app/node_modules/fastify/lib/handleRequest.js:129:5)\n    at validationCompleted (/app/node_modules/fastify/lib/handleRequest.js:119:5)\n    at preValidationCallback (/app/node_modules/fastify/lib/handleRequest.js:96:5)\n    at handler (/app/node_modules/fastify/lib/handleRequest.js:73:7)\n    at Object.handleRequest (/app/node_modules/fastify/lib/handleRequest.js:29:5)\n    at runPreParsing (/app/node_modules/fastify/lib/route.js:571:19)\n    at Object.routeHandler [as handler] (/app/node_modules/fastify/lib/route.js:491:7)"},"msg":"Invalid time value"}
api-1   | {"level":30,"time":1753386073522,"pid":18,"hostname":"7580ff6a71d1","reqId":"req-4","res":{"statusCode":500},"responseTime":0.855806827545166,"msg":"request completed"}
[Knginx exited with code 0
worker  | npm error path /app
worker  | npm error command failed
worker  | npm error signal SIGTERM
worker  | npm error command sh -c node dist/worker/worker.js
worker  | npm error A complete log of this run can be found in: /root/.npm/_logs/2025-07-24T19_40_17_292Z-debug-0.log
[Kworker exited with code 1
api-2   | npm error path /app
api-2   | npm error command failed
api-2   | npm error signal SIGTERM
api-2   | npm error command sh -c node dist/server.js
api-1   | npm error path /app
api-2   | npm error A complete log of this run can be found in: /root/.npm/_logs/2025-07-24T19_40_17_231Z-debug-0.log
api-1   | npm error command failed
api-1   | npm error signal SIGTERM
api-1   | npm error command sh -c node dist/server.js
api-1   | npm error A complete log of this run can be found in: /root/.npm/_logs/2025-07-24T19_40_17_231Z-debug-0.log
[Kapi-2 exited with code 1
[Kapi-1 exited with code 1
db      | 2025-07-24 19:41:20.161 UTC [1] LOG:  received fast shutdown request
redis   | 1:signal-handler (1753386080) Received SIGTERM scheduling shutdown...
redis   | 1:M 24 Jul 2025 19:41:20.190 * User requested shutdown...
redis   | 1:M 24 Jul 2025 19:41:20.190 * Saving the final RDB snapshot before exiting.
db      | 2025-07-24 19:41:20.200 UTC [1] LOG:  aborting any active transactions
db      | 2025-07-24 19:41:20.203 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
db      | 2025-07-24 19:41:20.203 UTC [63] LOG:  shutting down
db      | 2025-07-24 19:41:20.325 UTC [63] LOG:  checkpoint starting: shutdown immediate
redis   | 1:M 24 Jul 2025 19:41:20.329 * DB saved on disk
redis   | 1:M 24 Jul 2025 19:41:20.329 # Redis is now ready to exit, bye bye...
db      | 2025-07-24 19:41:20.857 UTC [63] LOG:  checkpoint complete: wrote 40 buffers (0.2%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.111 s, sync=0.213 s, total=0.654 s; sync files=13, longest=0.165 s, average=0.017 s; distance=183 kB, estimate=183 kB
db      | 2025-07-24 19:41:20.862 UTC [1] LOG:  database system is shut down
[Kredis exited with code 0
[Kdb exited with code 0
